src/app/dashboard/Missions/MissionEditView.jsx

```diff
@@ -226,14 +207,28 @@ const MissionEditView = ({ mission, toDetailsView, toListView, volunteers }) =>
     date: values.deliveryWindow.startTime,
     location: "",
   });
+  console.log("mission tentative:" + mission.tentativeVolunteerUid);
+  console.log(mission);
   const [selectedVolunteer, setSelectedVolunteer] = useState(
-    volunteers.find((el) => el.id === mission.volunteerUid)
+    volunteers.find(
+      (el) => el.id === mission.volunteerUid || el.id === mission.tentativeVolunteerUid
+    )
+  );
+  console.log("selected is ...");
+  console.log(selectedVolunteer);
+  const [selectedGroup, setSelectedGroup] = useState(
+    groups.find((el) => el.id === mission.groupUid)
   );
 
   const props = { classes, mission };
 
   const volunteerEditable = volunteerStatus.includes(mission.status);
 
+  useEffect(() => {
+    document.getElementById("pickup-address-id").value = values.pickUpLocation?.address;
+    document.getElementById("delivery-address-id").value = values.deliveryLocation?.address;
```

Comment:
Im not 100% sure I'm following the logic and reasoning to update this value.

I think what you're trying to achieve can be solved by by these elements setting their values based on state or props instead.

There aren't many cases where we _need_ to use document.getElementById. Majority of use cases can be solved with react element refs.

Reply:
If you're just trying to set the value of the address input I had previously added the `value` prop which you can use instead

Reply:
You're goign to have to spell it out for me.  I don't know how to set the default value of the AddressInputElement without doing the above.  Whatever you want me to insert, I'll do it.

Reply:
Ok, @qdozaq instructed me on the proper way to do this, so this shoudl also now be addressed now.